### ISSUE TITLE
12 hour 'timeFormat' on 'datetime' input does not show AM/PM options

### ISSUE THREAD

**Issue by JayWalker512** [Holt59/cakephp3-bootstrap-helpers#47]
Let me start by saying I'm super glad you made these helpers, they rock :)

I'm having an issue where I display an input for a 'datetime' field using 12 hour timeFormat parameter, but the displayed box only shows 12 hours and does not allow selection of AM or PM. Here is the code I'm using to display the input field:

```php
echo $this->Form->input('start_time', ['type' => 'datetime', 
                                                     'interval' => 5,
                                                     'minYear' => date("Y") - 1,
                                                     'maxYear' => date("Y"),
                                                     'timeFormat' => 12,
                                                     'label' => "Start Date and Time"]);
```

And here is an image of the produced output:

![datetime-12hour-timeformat-bug](https://cloud.githubusercontent.com/assets/1485718/9390180/805504ba-473d-11e5-8ddf-ad282f2c9a3e.png)

**Comment by Holt59** [comment]
Shoud be fixed with version `3.0.2`, simply update to the latest version:
```
composer clear-cache
composer update holt59/cakephp3-boostrap-helpers
```
Be careful since the name of the package has changed this afternoon (bootstrap3 -> bootstrap), you may want to remove the old package:
```
composer remove holt59/cakephp3-bootstrap3-helpers
composer require holt59/cakephp3-boostrap-helpers
```
And change `Bootstrap3` to `Bootstrap` where necessary (when loading the plugin or the helpers).

**Comment by JayWalker512** [comment]
That fixed it, thanks a bunch!
